### ISSUE TITLE
Test of problem submission broken

### ISSUE THREAD

**Issue by royragsdale** [picoCTF/picoCTF-web#10]
Currently the functional test: test_solve_getting_started fails.

This is because the Example challenges that are loaded by default no longer include a "Getting Started" Challenge.

A possible solution is re-create a simple "Getting Started" challenge as an example in [picoCTF-problems](https://github.com/picoCTF/picoCTF-problems) so that this test will serve it's expected functionality.

**Comment by cganas** [comment]
You are entirely correct! This feature was rushed out and not generalized. Ideally we should be querying for any given problem instead of assuming one exists.

**Comment by cganas** [comment]
With the resolution of #9, we can now easily run our broken test. :p

**Comment by royragsdale** [comment]
To continue the discussion of how best to test challenge submissions, perhaps it makes sense to exercise the full workflow and the concept of `solve.py` from the picoCTF-problems repo. This would likely result in the most general solution.

**Comment by cganas** [comment]
That is correct. Something that was felt strongly was that the functional tests could be ran remotely. As long as we maintain that invariant I'd like to get as close to the full stack workflow as possible.
